examples/MuscularSnake/get_connection_vector.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+# Join the two rods
+from elastica._linalg import (
+    _batch_norm,
+    _batch_matvec,
+)
+
+
+
+def get_connection_vector_straight_straight_rod(
```

Comment:
Don't we have this in some experimental module?

[here](https://github.com/GazzolaLab/PyElastica/blob/update-0.2.3/elastica/experimental/connection_contact_joint/parallel_connection.py)

Reply:
This is function computes the connection vector for specified index of rod one. I will update this [one](https://github.com/GazzolaLab/PyElastica/blob/update-0.2.3/elastica/experimental/connection_contact_joint/parallel_connection.py)